.mdlrc

```diff
@@ -0,0 +1,5 @@
+# a seperate "style" file must be used to pass "parameters" to a rule
```

Comment:
Typo: separate

Reply:
Argh.  I bet this exists in several repos from copying this file again.  Fixed.